examples/eks-msk-opensearch/README.md

```diff
@@ -0,0 +1,74 @@
+## Multi-Tenant logging using Fluent-bit, EKS , Amazon MSK and  Amazon OpenSearch
+
+In this example,we will showcase how to send your application logs from EKS's selective namespace(useful when you are using namesapce as tenant isolatio) and forward logs Amazon Managed Service for Kafka to store andto OpeneSearch for visualization. To achieve this, we will deploy Fluent Bit as a DaemonSet to tail /var/log/containers/*.log on the EKS cluster and use fluent-bit annotations to configure desired parser for pod's ina namespace enabled for logging via variable using terraform.This will create one topic for each namespace(enabled for logging) in Amazon MSK and then using MSK connector for OpenSearch we will send these logs to Opensearch such that each namespace(tenant) will have one Index.In the end there is a link to OpenSearch multi-tenancy configuration using RBAC from OpneSearch .
```

Comment:
space after comma after "example", and "namespace(useful". Please use the view menu in github to check how this text will look once its merged and rendered in markdown.

Reply:
> to tail /var/log/containers/*.log

I think in an intro is probably better to write in use case/product manager language, so I'd say "collect logs from pods" or something like that to indicate the use case for non-technical audience.

Reply:
> for pod's ina namespace

What does htis mean?

Reply:
> In the end there is a link to OpenSearch multi-tenancy configuration using RBAC from OpneSearch .

So I am picking out this sentence as an example to give a recommendation on how to improve your writing. 

Building on my previous comment, let's write the intro for a more PM/product/show me the use case value, the why not the how type of audience. 

Let's sell this tutorial. I recommend re-writing this intro paragraph to use language that clearly explains why someone should deploy this tutorial. The current writing style is tutorial style language, which works well once we get to the actual steps, but I'd write the intro differently.